packages/gatsby-plugin-layout/src/wrap-page.js

```diff
@@ -4,6 +4,10 @@ const preferDefault = m => (m && m.default) || m
 let Layout
 try {
   Layout = preferDefault(require(GATSBY_LAYOUT_COMPONENT_PATH))
+  if (process.env.NODE_ENV !== `production`) {
```

Comment:
!prod looks confusing, how about `=== dev`

Reply:
Fixed, thanks for your feedback @pranshuchittora